Games/types/Mafia/items/Snowball.js

```diff
@@ -17,11 +19,17 @@ module.exports = class Snowball extends Item {
                     item: this,
                     run: function() {
                         var reveal = this.item.reveal;
-
                         if (reveal == null)
                             reveal = Random.randArrayVal([true, false]);
 
-                        if (reveal)
+                        var cursed = this.item.cursed;
+                        if (cursed) {
+                            this.target = this.actor;
+                        }
+
+                        if (reveal && cursed)
```

Comment:
I did a test game w/ fab and both snowball + knife are not hitting this and are instead hitting the else, so it's just saying someone threw it at the person that is doing the action

Reply:
nice ok I'll check this 👍🏻👍🏻

Reply:
ok double checked this -- it will hit both cases